src/bin/stratisd.rs

```diff
@@ -118,7 +118,10 @@ fn trylock_pid_file() -> StratisResult<File> {
                     "Must be running as root in order to start daemon.".to_string(),
                 ));
             }
-            return Err(e.into());
+            return Err(StratisError::Error(format!(
```

Comment:
Could you just drop the special handling of ```PermissionDenied``` ErrorKind, while you're here? It's not a permanently accurate statement, since requirements might be modified from exactly "running as root" in some future, so it has potential to mislead. And the newly added error should allow users to figure out what went wrong from the representation of the IO error in it.

Reply:
Resolved.